spec/rubocop/formatter/base_formatter_spec.rb

```diff
@@ -44,10 +44,10 @@ def run
           runner.run([])
         end
 
-        describe 'invocation order' do
+        xdescribe 'invocation order' do
```

Comment:
I think we have to "eat our own dog food", meaning that we must find another way to implement the thing we used `method_missing` for here.